### ISSUE TITLE
Demos not working properly

### ISSUE THREAD

**Issue by Climax777** [dinony/od-virtualscroll#31]
Using chrome/firefox in Ubuntu 18.10 and macOSX (Mojave) at least two of the demo's have display issues which I opened here: https://github.com/dinony/od-vsadvanced/issues/1

https://imgur.com/xSgu3vt
https://imgur.com/a/bELQIaF

The issue seems to come from the od-virtualrow elements, wrapping over to the next row (basically two rows then), which breaks the first few columns of the next row. It gets exaggerated when zooming in (page zoom)

**Comment by Climax777** [comment]
Has there been any progress into this? Confirmations at least?

**Comment by dinony** [comment]
Yep, I've checked the issue on debian/chromium. From your screenshots I've assumed it's related to zooming. It looks like that on debian/ubuntu the browser does not fire a window resize event, when zoomed.

Could you tell me whether you have the same bug with initial zoom: 100%?

**Comment by Climax777** [comment]
> Could you tell me whether you have the same bug with initial zoom: 100%?

It happens on the advanced example on load with 100% zoom too, on my large 4k screen. On smaller screens it happens when zooming in.

Just tested again on all 3 platforms (Windows, Linux, mac) with chrome and firefox.

**Comment by Climax777** [comment]
This only happens with flexbox. I think the scrollbar's width also plays a role. I can "fix" it by disabling flexbox in the demo container, but then a horizontal scrollbar appears with a very small off-screen area to the right.
